traffic_ops/traffic_ops_golang/README.md

```diff
@@ -125,11 +125,11 @@ Now we need to create the Go endpoint.
 
 #### Getting a "Handle" on Routes
 
-Open [routes.go](https://github.com/apache/trafficcontrol/blob/master/traffic_ops/traffic_ops_golang/routes.go). Routes are defined in the `Routes` function, of the form `{version, method, path, handler}`. Notice the path can contain variables, of the form `/{var}/`. These variables will be made available to your handler.
+Open [routes.go](https://github.com/apache/trafficcontrol/blob/master/traffic_ops/traffic_ops_golang/routing/routes.go). Routes are defined in the `Routes` function, of the form `{version, method, path, handler}`. Notice the path can contain variables, of the form `/{var}/`. These variables will be made available to your handler.
```

Comment:
Sorry, I think I missed this on my last review. This could be a relative URL as well:

```markdown
Open [routes.go](./routing/routes.go). Routes are defined in the `Routes` function...
```

I also don't think I ever said this, but the advantage of that being that until the folder structure changes, this will always point at the correct network location, no matter what revision you're looking at. Like if you switch to a new branch, the full URL would redirect you to master, but the relative one will move you without changing branches.